src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py

```diff
@@ -124,21 +124,150 @@ class PerformanceLattice(BaseModel):
+            as measured by a pi-wait-pi protocol (measured in s)
+        T1Ensemble (Decimal): Lifetime of the Rydberg state for an ensemble of non-interacting qubits distributed over
+            the user region, in the absense of drive, as measured by a pi-wait-pi protocol (measured in s)
+        T2StarSingle (Decimal): Typical dephasing time of a single non-interacting qubit in the absense of drive, as measured by
+            a Ramsey protocol. Includes coherent and incoherent processes. (measured in s)
+        T2StarEnsemble (Decimal): Dephasing time of an ensemble of non-interacting qubits distributed over the user region, in
+            the absense of drive, as measured by a Ramsey protocol. Includes coherent and incoherent processes. (measured in s)
+        T2EchoSingle (Decimal): Typical dephasing time of a single non-interacting qubit in the absense of drive, as measured by a
+            spin-echo dynamical decoupling protocol. This measurement isolates the effects of incoherent processes (measured in s)
+        T2EchoEnsemble (Decimal): Dephasing time of an ensemble of non-interacting qubits distributed over the user region, in the absense
+            of drive, as measured by a spin-echo dynamical decoupling protocol. This measurement isolates the effects
+            of incoherent processes. (measured in s)
+        T2RabiSingle (Decimal): Typical decoherence time of a single driven qubit, as measured by a Rabi oscillation protocol with variable
+            pulse duration a maximum Rabi frequency. Includes coherent and incoherent processes.(measured in s)
+        T2RabiEnsemble (Decimal): Decoherence time of an ensemble of non-interacting driven qubits distributed over the user region, as measured
+            by a Rabi oscillation protocol with variable pulse duration at maximum Rabi frequency. Includes
+            coherent and incoherent processes. (measured in s)
+        T2BlockadedRabiSingle (Decimal): Typical decoherence time of a single pair of driven blockaded qubits, as measured by a Rabi oscillation
+            protocol with variable pulse duration a maximum Rabi frequency. Includes coherent and incoherent processes. (measured in s)
+        T2BlockadedRabiEnsemble (Decimal): Decoherence time of an ensemble of pairs of driven blockaded qubits distributed over the user region
+            (different pairs do not interact with each other), as measured by a Rabi oscillation protocol with variable pulse duration at
+            maximum Rabi frequency. Includes coherent and incoherent processes. (measured in s)
+        detuningError (Decimal): Systematic error from specified value of the global detuning averaged over the user
+            region. (measured in rad/s)
+        detuningInhomogeneity (Decimal): RMS inhomogeneity of the detuning over the user region. (measured in rad/s)
+        rabiAmplitudeRampCorrection (List[RabiCorrection]): dynamic correction curve of effective single-qubit
```

Comment:
```suggestion
        rabiAmplitudeRampCorrection (list[RabiCorrection]): dynamic correction curve of effective single-qubit
```
Sorry for randomly jumping into the review. It might be good to be consistent with the comment below about using built-in type.

Reply:
Agreed. I will be opening a separate PR to make use of these types and for the type being, my suggestion will be edited to use the typing import.

Reply:
@tachikoma-li not random at all, thanks for your suggestion. Will default to using List from typing for the scope of this PR.